dists/amigaos/Ext_Inst_so.rexx

```diff
@@ -1,153 +0,0 @@
-/*
```

Comment:
I would prefer that the history is kept, so instead of removing the file and then adding a copy with modifications, you could do `git mv <form> <to>`, then modify the file, then commit.

This will keep the history and also show the diff nicely.

Reply:
the local git command i can use (based on a port of sgit) is *very* limited and i don't think it's possible from the web interface, is it?

is there some other way i could tread to keep the history? (which I'd not like to lose too)

Reply:
Yes, the file name is editable when you edit in-place:
<img width="416" alt="Screenshot 2024-05-04 at 00 21 06" src="https://github.com/scummvm/scummvm/assets/365188/e3892a4b-5dcd-44aa-9fab-024267167217">

Reply:
ah...i see

will have to redo the pr then...not today though

thank you for the pointers